vmprof/stats.py

```diff
@@ -14,10 +14,12 @@ def __init__(self, profiles, adr_dict=None, jit_frames=None, interp=None,
         if state:
             self.profile_lines = state.profile_lines
             self.profile_memory = state.profile_memory
+            self.adr_size = len(state.virtual_ips)
         else:
             # unkown, for tests only
-            self.profile_lines = False
-            self.profile_memory = False
+            self.profile_lines = None
+            self.profile_memory = None
+            self.adr_size = None
```

Comment:
Ah, that particular thing I added literally only for one test, I suppose we could rework that test instead to use a lower-level API?